spec/controllers/user_controller_spec.rb

```diff
@@ -16,7 +16,6 @@
 
       put :update, user: {email: "user@example.com",
         current_password: "password", bio: "I updated my profile"}
```

Comment:
Stylistic issue here. I see why you brought `current_password` down to the next line (seeing that the line was getting really long. But instead you should probably take it another step further.
```
put :update, user: {
  email: "user@example.com",
  current_password: "password", 
  bio: "I updated my profile"
}
```

Again, good on recognizing that the line was getting too long and making a move to fix that.